### PR TITLE
add guidance for pfx to pem conversion

### PR DESCRIPTION
**Pull request by derisen**
This PR adds some tips for converting `pfx` formatted certificates to `pem` to msal-node certificates documentation.

### CODE REVIEW

File: lib/msal-node/docs/key-vault-managed-identity.md [hunk 1]
```diff
@@ -151,6 +95,59 @@ main();
...
 
+> :information_source: Converting `PKCS12/PFX` to `PEM`
+>
+> In most circumstances, Azure Key Vault can export certificates and private keys in `pem` format (see: [Export stored certificates](https://docs.microsoft.com/azure/key-vault/certificates/how-to-export-certificate?tabs=azure-cli#export-stored-certificates)), if **Content Type** was chosen as `pem` during certificate generation (see: [Create a certificate in Key Vault](https://docs.microsoft.com/azure/key-vault/certificates/tutorial-rotate-certificates#create-a-certificate-in-key-vault)). If for some reason this is not the case, OpenSSL can be used for conversions:
+>
+> ```bash
+> openssl pkcs12 -in certificate.pfx -out certificate.pem
+> ```
+>
+> If the conversion needs to happen programmatically, then you may have to rely on a 3rd party package, as Node.js offers no native method for this. For instance, using a popular TLS implementation like [node-forge](https://www.npmjs.com/package/node-forge), you can do:
```
Comment (jasonnutter) [inline lib/msal-node/docs/key-vault-managed-identity.md:106]: Seems like this doesn't need to be indented?

Comment (jasonnutter) [reply lib/msal-node/docs/key-vault-managed-identity.md:106]: And are there any other libraries besides `node-forge` we can recommend?

Comment (sameerag) [reply lib/msal-node/docs/key-vault-managed-identity.md:106]: I believe we did not find any. Any recommendations?

Comment (derisen) [reply lib/msal-node/docs/key-vault-managed-identity.md:106]: @jasonnutter @sameerag I've found a dozen small pfx to pem packages, but none very popular. [pem](https://www.npmjs.com/package/pem) is a good one that I tried and works fine. Haven't tried [google-p12-pem](https://www.npmjs.com/package/google-p12-pem) but looks promising. Shall we mention these?

Comment (derisen) [reply lib/msal-node/docs/key-vault-managed-identity.md:106]: Looks like **google-p12-pem** is using node-forge under the hood. Will give [jose](https://www.npmjs.com/package/jose) a try

**Review by jasonnutter — APPROVED** [review]

**Comment by derisen** [comment]
@sameerag @hectormmg @samuelkubai fyi this is ready for review

**Review by hectormmg — APPROVED** [review]
Looks great, thanks @derisen !

**Pull request merged**